functions/Invoke-DbaDbDataGenerator.ps1

```diff
@@ -244,12 +248,18 @@ function Invoke-DbaDbDataGenerator {
 
                                         # Generate a new value
                                         try {
+                                            # TODO: Why is this set here?
```

Comment:
I think that is a remnant of previous versions of the command and can be removed.
Later in the command this is needed though because that variable is used heavily to set the new random values.